File: structure/move.js


const move = arr => {
    let first = arr.slice(0, 4).concat(arr.slice(5));
    let result = first.slice(0, 2).concat([arr[4]]).concat(first.slice(2)).concat([arr[4]]);

    return result;
}

let a = [34, 54, 67, 89, 11, 43, 94];


console.log(move(a));

File: structure/move.py


def move(arr):
    first = arr[:4] + arr[5:]
    result = first[:2] + [arr[4]] + first[2:] + [arr[4]]

    return result 


List = [34, 54, 67, 89, 11, 43, 94]

print(move(List))